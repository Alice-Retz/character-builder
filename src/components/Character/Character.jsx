import React from "react";

export default function Character({ head, middle }) {
	return (
		<section style={{ display: "flex", flexDirection: "column" }}>
			<article
				className='head'
				style={{
					backgroundImage: `url(./${head}-head.png)`,
					backgroundSize: "cover",
					backgroundRepeat: "no-repeat",
					height: 170,
					width: 170,
				}}></article>
			<article
				className='middle'
				style={{
					backgroundImage: `url(./${middle}-middle.png)`,
					backgroundSize: "cover",
					backgroundRepeat: "no-repeat",
					height: 170,
					width: 150,
				}}></article>
		</section>
	);
}
