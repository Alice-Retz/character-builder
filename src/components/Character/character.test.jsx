import { render } from "@testing-library/react";
import Character from "./Character";

it("should render the Character component", () => {
	const { container } = render(
		<Character head='boar' middle='blue' bottom='blue' />
	);
	expect(container).toMatchSnapshot();
});
