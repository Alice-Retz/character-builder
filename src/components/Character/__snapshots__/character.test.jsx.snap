// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Character component 1`] = `
<div>
  <section
    style="display: flex; flex-direction: column;"
  >
    <article
      class="head"
      style="background-image: url(./boar-head.png); background-size: cover; background-repeat: no-repeat; height: 170px; width: 170px;"
    />
    <article
      class="middle"
      style="background-image: url(./blue-middle.png); background-size: cover; background-repeat: no-repeat; height: 170px; width: 150px;"
    />
    <article
      class="bottom"
      style="background-image: url(./blue-bottom.png); background-size: cover; background-repeat: no-repeat; height: 300px; width: 170px;"
    />
  </section>
</div>
`;
